extensions/GitExtension/FileExplorerGitIntegration/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using FileExplorerGitIntegration.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Windows.AppLifecycle;
+using Serilog;
+using Windows.ApplicationModel.Activation;
+
+namespace FileExplorerGitIntegration;
+
+public sealed class Program
+{
+    [MTAThread]
+    public static void Main([System.Runtime.InteropServices.WindowsRuntime.ReadOnlyArray] string[] args)
+    {
+        // Set up Logging
+        Environment.SetEnvironmentVariable("DEVHOME_LOGS_ROOT", Path.Join(DevHome.Common.Logging.LogFolderRoot, "FileExplorerGitIntegration"));

```

Comment:
NIT: You can use Microsoft.Windows.System.EnvironmentManager and the EV will be removed when the app is uninstalled.